### PR TITLE
Required capabilities validation

### PR DESCRIPTION
**Pull request by kkohbrok**

### CODE REVIEW

File: openmls/src/group/core_group/test_proposals.rs [hunk 1]
```diff
@@ -447,7 +447,7 @@ fn test_group_context_extensions(
     .expect("Error joining group.");
 }
 
-#[apply(ciphersuites_and_backends)]
+//#[apply(ciphersuites_and_backends)]
```
Comment (sgmenda) [inline openmls/src/group/core_group/test_proposals.rs:450]: This seems like a typo?

Comment (kkohbrok) [reply openmls/src/group/core_group/test_proposals.rs:450]: Good catch! It kind of is a typo. I regularly comment out these macro invocations so `rust-analyzer` is a little more helpful when working on tests. I usually try not to commit these, but it seems like this one slipped through. Then again, this is not meant to be merged just yet.

Comment (sgmenda) [reply openmls/src/group/core_group/test_proposals.rs:450]: Oops, I didn't notice the `draft` status. 😬 Yup, ++rust-analyzer.

**Comment by kkohbrok** [comment]
I think it makes sense to wait with this until we deal with #701.

**Pull request closed without merging**